backend/src/main/java/de/jmpsoftware/backend/service/DMXService.java

```diff
@@ -230,7 +234,13 @@ public FixtureDB getFixtureFromList(String name) {
 
         FixtureDB fixtureDB = getFixtureFromList( name );
 
-        fixtureDB.getFaderList().forEach(fader -> tmpList.add(getFaderFromFixtureFaderList(fader, fixtureDB.getAddress(), fixtureDB.getUniverse(), name)));
+        fixtureDB.getFaderList().forEach(fader -> {
+            try {
+                tmpList.add(getFaderFromFixtureFaderList(fader, fixtureDB.getAddress(), fixtureDB.getUniverse(), name));
+            } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
Are you sure that you want to continue like nothing happend when a error appears? Im not sure what your data flow is like and what makes snese here. 

I would log the exception via a logger